packages/sdk/src/next/types/VariableTypes.ts

```diff
@@ -1,8 +1,10 @@
-import { ListVariableItem, Variable } from '../../types/VariableTypes';
+import { ListVariableItem, ValueWithStyle, Variable } from '../../types/VariableTypes';
 
 export interface ListVariable extends Variable {
     items: ListVariableItem[];
     selected?: ListVariableItem;
+    prefix?: ValueWithStyle;
+    suffix?: ValueWithStyle;
```

Comment:
```suggestion
    selected?: ListVariableItem | null;
    prefix?: ValueWithStyle | null;
    suffix?: ValueWithStyle | null;
```
I think it will work anyway but I'd change it for explicitness here and in the other new models https://github.com/chili-publish/studio-sdk/pull/480#discussion_r1673664102